packages/common/src/core/schemas/shared/HubItemEntitySchema.ts

```diff
@@ -29,6 +29,11 @@ export const HubItemEntitySchema: IConfigurationSchema = {
     tags: ENTITY_TAGS_SCHEMA,
     categories: ENTITY_CATEGORIES_SCHEMA,
     isDiscussable: ENTITY_IS_DISCUSSABLE_SCHEMA,
+    // Do not remove, read-only property for validation rules
```

Comment:
can both of these be removed now?

Reply:
Yes, thanks for catching that. I thought I'd already removed everything